.github/workflows/gallery.yml

```diff
@@ -0,0 +1,61 @@
+name: Gallery
+
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+
+    - name: Install Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.x'
+
+    - uses: actions/cache@v3
+      name: Cache/restore pip data
+      with:
+        path: ~/.cache/pip
+        key: pip
+
+    - name: Install Python dependencies
+      run: pip install -r requirements.txt
+
+    - name: Cache/restore Playwright browsers
+      uses: actions/cache@v3
+      with:
+        path: ~/.cache/ms-playwright/
+        key: playwright
+
+    - name: Install Playwright dependencies
+      run: shot-scraper install
+
+    - name: Build everything
+      run: python build_gallery.py
+
+    - name: Upload artifact
+      uses: actions/upload-pages-artifact@v2
+      with:
+        path: gallery
+
+  deploy:
```

Comment:
Careful review needed here: I'm not sure deploying to GitHub pages is desirable, given MkDocs documentation is served under its own custom domain.

Reply:
Deploying it like that would be the same as pushing to a `gh-pages` branch here, meaning it would be under `https://mkdocs.org/catalog`, assuming the deployment happens on this repository, which is what I assume.

Reply:
Thanks, that's what I'm suspecting too. @tomchristie can you confirm that docs pushed on the gh-pages branch of mkdocs/catalog will end up at www.mkdocs.org/catalog?